nautobot/dcim/templates/dcim/device.html

```diff
@@ -426,5 +426,27 @@
         </div>
     </div>
 {% endif %}
+{% if object.is_metadata_associable_model and perms.extras.view_objectmetadata %}
```

Comment:
Do we need this in any other specific templates? ISTR the software-version or software-image-file templates had some similar issues.

Reply:
Yes! `softwareimagefile_retrieve.html` thank you for catching that.